Libraries/CMSIS/Device/Maxim/MAX32672/Include/max32672.h

```diff
@@ -307,7 +307,7 @@ mxc_aes_key_regs_t mxc_sys_aes_key_regs_t;
 
 /******************************************************************************/
 /*                                                                        AES */
-#define MXC_BASE_AES ((uint32_t)0x40007400UL)
+#define MXC_BASE_AES ((uint32_t)0x40207400UL)
```

Comment:
This is not something that changed for B2 is it?  All revisions should be using this address?

Reply:
Jacob confirmed this change applies to all revs.